### PR TITLE
Us537 A - Add configuration properties for IRuleset: As a developer I want all configurable Monopoly rules to be defined in the RuleSet interface so that different rule configurations can be swapped.

### PR DESCRIPTION
**Pull request by kescob0825**

### CODE REVIEW

File: Assets/Scripts/Managers/Rules/StandardRuleSet.cs [hunk 1]
```diff
@@ -19,16 +19,44 @@ public class StandardRuleSet : IRuleSet
         [Tooltip("Utility multiplier when owner has both utilities.")]
         [SerializeField] private int utilityBoth = 10;
 
+        [Tooltip("Starting money for each player.")]
+        [SerializeField] private int playerStartingMoney = 1500;
+
+        [Tooltip("Salary received when passing GO.")]
+        [SerializeField] private int gOSalary = 200;
```
Comment (hdathert) [inline Assets/Scripts/Managers/Rules/StandardRuleSet.cs]: Typo, should be `goSalary`

**Review by hdathert — CHANGES_REQUESTED** [review]
Small typo needs fixing, otherwise looks good.

**Review by dzadroga — APPROVED** [review]
# Code Review Checklist

**Completed for**: US #537 and Tasks #539 to #548
**Code Review Completed By**: Dakota Zadroga

Use this checklist to review new code changes before merging.

---

## Implementation
- [X] Does this code change accomplish what it is supposed to do?
- [X] Can the solution be simplified?
- [X] This change does **not** add unwanted compile-time or run-time dependencies.
- [X] The code is at the right abstraction level.
- [X] The code is modular enough.
- [X] The solution is adequate for maintainability, readability, performance, and security.
- [X] Similar functionality does **not** already exist in the codebase (or reuse is not appropriate).
- [X] No additional best practices or design patterns would substantially improve this code.

---

## Logic Errors and Bugs
- [X] No use cases where the code behaves incorrectly.
- [X] No inputs or external events that can break the code.
- [X] Code agrees with specifications.
- [X] No off-by-one errors.
- [X] No magic numbers (or they are well-named constants).
- [X] Proper use of global/large-scope variables (minimized and justified).

---

## Error Handling and Logging
- [X] Error handling is implemented correctly.
- [X] Error messages are user-friendly (where applicable).
- [X] Sufficient, meaningful log events exist for debugging (no sensitive data in logs).

---

## Dependencies
- [X] Documentation/configuration/README updates were made as needed.
- [X] No negative impact on other parts of the system or backward compatibility.

---

## Security and Data Privacy
- [X] No new security vulnerabilities introduced.
- [X] All user input is validated, sanitized, and escaped (XSS/SQL injection prevented).
- [X] Data from external APIs/libraries is handled safely and checked as needed.

---

## Performance
- [X] No significant performance regressions.
- [X] No obvious opportunities for significant performance gains left unaddressed.

---

## Ethics and Morality
- [X] Appropriate measures exist to prevent or report harassment/abuse (if applicable).

---

## Testing and Testability
- [X] Code is testable.
- [X] Automated tests have been added or updated.
- [X] Existing tests sufficiently cover the change (unit/integration/system).
- [X] Edge cases and boundary inputs are tested.

---

## Readability
- [X] Code is easy to understand.
- [X] Methods/functions are appropriately small and focused.
- [X] Names (functions/classes/variables) are clear and descriptive.
- [X] Code is in the correct file/folder/package.
- [X] Control flow is intuitive; data flow is clear.
- [X] Comments are clear, relevant, and up to date.
- [X] No commented-out or redundant code remains.
- [X] No overly dense one-liners; no overly long methods.

---

**Use this before submitting a pull request.**  
It helps ensure code quality, maintainability, and compliance with best practices.

*Based on awesome code review guides and MIT’s code review recommendations.*

**Once other change is made - for renaming - PR looks good to me and can be merged.**
